solr/bin/solr

```diff
@@ -1446,9 +1446,6 @@ if [ $# -gt 0 ]; then
   done
 fi
 
-# Setting number of threads for search
-if ! command -v nproc &> /dev/null; then echo "Couldn't determine number of CPUs, using default number of search threads"; else cpus=`nproc`; SCRIPT_SOLR_OPTS+="-Dsolr.searchThreads=$cpus"; fi
-
```

Comment:
@chatman's https://github.com/apache/solr/commit/8299cf3f9a6763d1d728651ad3f2e53fa696a346 commit from the https://github.com/apache/solr/commits/jira/solr-13350-fixes-9x/ branch.